systems/drone/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+INPUTS="/tmp/resim/inputs"
+OUTPUTS="/tmp/resim/outputs"
+
+
+# Apply any parameters needed to the experience
+
+c=`cat <<EOF
```

Comment:
it is a bit unfortunate that we mix the build/experience params concept here... nothing for it for now